packages/web-api-scan-runner/src/runner/runner.spec.ts

```diff
@@ -340,6 +382,74 @@ describe(Runner, () => {
         loggerMock.verifyAll();
     });
 
+    describe('enqueue notification, send notification feature flag is enabled', () => {
+        const scanNotifyUrl = 'notify-url';
+        const notificationMessage: OnDemandNotificationRequestMessage = {
+            scanId: onDemandPageScanResult.id,
+            scanNotifyUrl: scanNotifyUrl,
+            runStatus: 'completed',
+            scanStatus: 'pass',
+        };
+
+        beforeEach(() => {
+            setupWebDriverCalls();
+
+            scannerTaskMock
+                .setup(async s => s.scan(scanMetadata.url))
+                .returns(async () => Promise.resolve(passedAxeScanResults))
+                .verifiable();
+
+            setupGenerateReportsCall(passedAxeScanResults);
+            setupSaveAllReportsCall();
+
+            serviceConfigurationMock.reset();
+            const featureFlags1: FeatureFlags = { sendNotification: true };
+            serviceConfigurationMock
+                .setup(async scm => scm.getConfigValue('featureFlags'))
+                .returns(async () => Promise.resolve(featureFlags1))
+                .verifiable(Times.once());
+        });
+
+        it('Notification URL is not null', async () => {
```

Comment:
you can merge the two tests to one, using test.each since both have the same code, except for the notification object, which can be passed as a test case using test.each